Updated items in dayzadmin and GIP, fixed binds and improved reloading.

Loki214 opened the pull request:
Binds are disabled when editing in dayzadmin , GIP and Journal.

Loki214 committed:
Update reload_player.lua

Loki214 committed:
Disabled the inventory when dead

Loki214 committed:
Updated items in dayzadmin and GIP.

Loki214 committed:
Fixed error in journal

mtadayz commented:
Now, the branch is conflicting is dayzadmin, so I'd say you remove your changes done in dayzadmin (store them somewhere), update the pull request, update your repository, then make your dayzadmin changes again and create another pull request.

mtadayz commented:
Now, the branch is conflicting is dayzadmin, so I'd say you remove your changes done in dayzadmin (store them somewhere), update the pull request, update your repository, then make your dayzadmin changes again and create another pull request.

Loki214 committed:
Improved reload and ammo hud

Blocked reloading weapons when it it already full.
Blocked reloading weapon when there is no extra ammo to reload.
Hud now shows exact reloading ammo.

Loki214 commented:
Ok
